### ISSUE TITLE
staging server: 'Please upgrade your ACME client to a version that supports ACMEv2'

### ISSUE THREAD

**Issue by mbican** [natemcmaster/LetsEncrypt#22]
```
fail: McMaster.AspNetCore.LetsEncrypt.Internal.AcmeCertificateLoader[0]
      Failed to create certificate
Certes.AcmeRequestException: Fail to load resource from 'https://acme-staging.api.letsencrypt.org/acme/new-reg'.
urn:acme:error:unauthorized: Account creation on ACMEv1 is disabled. Please upgrade your ACME client to a version that supports ACMEv2 / RFC 8555. See https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430 for details.
   at Certes.AcmeClient.ThrowIfError[T](AcmeResponse`1 response, Uri uri)
   at Certes.AcmeClient.NewRegistraton(String[] contact)
   at McMaster.AspNetCore.LetsEncrypt.Internal.CertificateFactory.RegisterUserAsync(CancellationToken cancellationToken) in c:\Users\bican\git\to-do-list-react\LetsEncrypt\src\LetsEncrypt\Internal\CertificateFactory.cs:line 51
   at McMaster.AspNetCore.LetsEncrypt.Internal.AcmeCertificateLoader.GetOrCreateCertificate(CertificateFactory factory, CancellationToken cancellationToken) in c:\Users\bican\git\to-do-list-react\LetsEncrypt\src\LetsEncrypt\Internal\AcmeCertificateLoader.cs:line 154
   at McMaster.AspNetCore.LetsEncrypt.Internal.AcmeCertificateLoader.LoadCerts(CancellationToken cancellationToken) in c:\Users\bican\git\to-do-list-react\LetsEncrypt\src\LetsEncrypt\Internal\AcmeCertificateLoader.cs:line 124
```

**Comment by mbican** [comment]
https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430

In November of 2019 we will stop allowing new account registrations through our ACMEv1 API endpoint

**Comment by mbican** [comment]
I'm taking this issue :-)

**Comment by natemcmaster** [comment]
Yikes, didn't realize they were shutting it down so soon. As soon as this is implemented, let's get a release out to NuGet.org. I'm going to pause work on all other enhancements to make sure we get this released ASAP.

**Comment by Bobchev** [comment]
@natemcmaster https://youtu.be/lrJj3En-M5g?t=29

**Comment by natemcmaster** [comment]
Resolved with #23

**Comment by Bobchev** [comment]
![great-teacher-you-are-success-you-will-have](https://user-images.githubusercontent.com/12601873/67017555-9eebe880-f102-11e9-9901-5e9b4ef62b9a.jpg)
